src/arcade/patch/agent/process/PatchProcessInflammation.java

```diff
@@ -0,0 +1,333 @@
+package arcade.patch.agent.process;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import ec.util.MersenneTwisterFast;
+import arcade.core.env.location.Location;
+import arcade.core.sim.Simulation;
+import arcade.core.util.Parameters;
+import arcade.core.util.Solver;
+import arcade.core.util.Solver.Equations;
+import arcade.patch.agent.cell.PatchCell;
+import arcade.patch.agent.cell.PatchCellCART;
+import arcade.patch.env.lattice.PatchLattice;
+
+/**
+ * Implementation of {@link Process} for inflammation type modules in which IL-2 is taken up and
+ * cytotoxic/stimulatory functions are modified.
+ *
+ * <p>The {@code Inflammation} module represents an 8-component signaling network.
+ */
+public abstract class PatchProcessInflammation extends PatchProcess {
+
+    /** Number of components in signaling network. */
+    protected static final int NUM_COMPONENTS = 8;
+
+    /** ID for IL-2, bound total. */
+    protected static final int IL2_INT_TOTAL = 0;
+
+    /** ID for IL-2, external. */
+    protected static final int IL2_EXT = 1;
+
+    /** ID for IL-2 receptors, total between both two and three chain complex. */
+    protected static final int IL2R_TOTAL = 2;
+
+    /** ID for two-chain IL-2 receptor complex. */
+    protected static final int IL2RBG = 3;
+
+    /** ID for three-chain IL-2 receptor complex. */
+    protected static final int IL2RBGa = 4;
+
+    /** ID for IL-2-two-chain IL-2 receptor complex. */
+    protected static final int IL2_IL2RBG = 5;
+
+    /** ID for IL-2-three-chain IL-2 receptor complex. */
+    protected static final int IL2_IL2RBGa = 6;
+
+    /** ID for granzyme, internal. */
+    protected static final int GRANZYME = 7;
+
+    /** Number of steps per second to take in ODE. */
+    private static final double STEP_DIVIDER = 3.0;
+
+    /**
+     * Rate of conversion of IL-2R two-chain complex to IL-2R three chain complex [/sec/step
+     * divider].
+     */
+    private static final double K_CONVERT = 1e-3 / STEP_DIVIDER;
+
+    /**
+     * Rate of recycling of receptor complexes back to IL-2 receptor two chain complex [/sec/step
+     * divider].
+     */
+    private static final double K_REC = 1e-5 / STEP_DIVIDER;
+
+    /** Rate of IL-2 binding to two-chain IL-2 receptor complex [um^3/molecules IL-2/min]. */
+    private double iL2BindingOnRateMin = 3.8193E-2;
```

Comment:
not sure this is a better solution - rateIL2BindingMin seems a little more readable to me